src/Menu.java

```diff
@@ -0,0 +1,140 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.Scanner;
+
+public class Menu {
+
+
+    public static void main(String[] args){
+
+        try {
+            int userInput;
+            do {
+                Utils.clearScreen();
+                BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+                Utils.println("starting_menu",1,2,3,4, Game.isSoundOn()? "Off": "On",5,6);
+                userInput = Integer.parseInt(reader.readLine());
+                switch(userInput){
+                    case(1):
+                        Game.play();
+                        break;
+                    case(2):
+                        //Learn about the trail
+                        learnAboutTrail();
+                        break;
+                    case(3):
+                        //See the top 10
+                        oregonTopTen();
+                        break;
+                    case(4):
+                        //Turn sound off
+                        Game.setSoundOn(!Game.isSoundOn());
+                        if(Game.isSoundOn()){
+                            System.out.println("Sound is now on");
+                        } else{
+                            System.out.println("Sound is now off");
+                        }
+                        break;
+                    case(5):
+                        //Choose Management Options
+                        managementMenu();
+                        break;
+                    case(6):
+                        // Exit the game
+                        //System.exit(0);
+                }
+            } while (userInput!=6);
+        } catch (IOException e) {
+            System.out.println("please enter a valid number");
+
+        }
+
+    }
+
+    public static void learnAboutTrail(){
+        Scanner scanner = new Scanner(System.in);
+        Utils.clearScreen();
+        Utils.println("learn_about_trail");
+        System.out.println("Press Enter to return to the menu..");
+        scanner.nextLine();
+    }
+
+    public static void oregonTopTen(){
+        //TODO: print top ten
+        Utils.println("oregon_top_ten");
```

Comment:
This method does not work
```suggestion
        Utils.println("oregon_top_ten");
        Keyboard.nextLine();
```